Game/Resources/final_event/MainScript.as

```diff
@@ -61,7 +61,7 @@ class MainScript {
         switch (phase) {
             case 0: { // Entering final scene.
                 vec3 pos = minecart.GetWorldPosition();
-                pos.x += speed * deltaTime;
+                pos.x -= speed * deltaTime;
```

Comment:
I don't see that the scene has been rotated in the superscene? Is this a leftover from testing locally?

Reply:
No this is needed in superscene